src/features/PassSurvey/model/AnswersConstructService.ts

```diff
@@ -135,6 +135,7 @@ export default class AnswersConstructService implements ICompletionConstructServ
         width: window.innerWidth,
         height: window.innerHeight,
       },
+      eventHistoryId: `${this.event.id}_${this.event.version}`,
```

Comment:
The chance of this happening may be unlikely, but just to be safe we should probably make sure both of these are here before including this property
```suggestion
      ...(this.event.version && { eventHistoryId: `${this.event.id}_${this.event.version}` }),
```